Description: Use local shared javascript libraries
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: no
Last-Update: 2017-04-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/docs/v1/index.html
+++ b/docs/v1/index.html
@@ -3230,8 +3230,8 @@
 
 </script>
 
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-<script src="browser-compiler/coffee-script.js"></script>
+<script src="js/jquery/jquery.js"></script>
+<script src="js/coffeescript/coffee-script.js"></script>
 
 </body>
 </html>
--- a/docs/v1/test.html
+++ b/docs/v1/test.html
@@ -3,8 +3,8 @@
 <head>
   <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
   <title>CoffeeScript Test Suite</title>
-  <script src="browser-compiler/coffee-script.js"></script>
-  <script src="https://cdn.jsdelivr.net/underscorejs/1.8.3/underscore-min.js"></script>
+  <script src="../js/coffeescript/coffee-script.js"></script>
+  <script src="../js/underscore/underscore.js"></script>
   <style>
     body, pre {
       font-family: Consolas, Menlo, Monaco, monospace;
